src/sentry/static/sentry/app/utils/log_exposure.jsx

```diff
@@ -0,0 +1,11 @@
+import HookStore from 'app/stores/hookStore';
+
+/**
+ * @param {String} name name of the experiment
+ * @param {Object} data Experiment data to be recorded: {unit_name, unit_id, params}
+ * 					note that params need to string
+ */
+export default function log_exposure(name, data) {
```

Comment:
Can you rename this (+ file) to be `camelCase`?

Reply:
ugh pythonBrain- thanks